tests/composite_viscous_outputs/screen-output

```diff
@@ -3,17 +3,17 @@ Loading shared library <./libcomposite_viscous_outputs.debug.so>
 
 * Connecting signals
 temperature (K)   eta (Pas)   creep stress (Pa)   edot_ii (/s)   edot_ii fractions (diff, disl, prls, drpr, max)
-1000 2.51996e+19 5.01993e+08 1e-11 9.45754e-07 0.00119879 0.00251493 0.996285 2.51996e-09
-1100 2.51513e+19 5.01025e+08 1e-11 6.08307e-05 0.456037 0.0313917 0.512511 2.51513e-09
-1200 7.70568e+18 1.52114e+08 1e-11 0.000594407 0.999399 6.44893e-06 0 7.70568e-10
-1300 2.39282e+18 4.58564e+07 1e-11 0.00338081 0.996619 3.19443e-09 0 2.39282e-10
-1400 9.18168e+17 1.63634e+07 1e-11 0.0149609 0.985039 1.26589e-11 0 9.18168e-11
-1500 4.32083e+17 6.64167e+06 1e-11 0.0538305 0.94617 2.09885e-13 0 4.32083e-11
-1600 2.47246e+17 2.94493e+06 1e-11 0.161077 0.838923 8.74429e-15 0 2.47246e-11
-1700 1.67379e+17 1.34758e+06 1e-11 0.397345 0.602655 5.42942e-16 0 1.67379e-11
-1800 1.28447e+17 568948 1e-11 0.749975 0.250025 2.2682e-17 0 1.28447e-11
-1900 1.09563e+17 191256 1e-11 0.962698 0.0373021 2.59166e-19 0 1.09563e-11
-2000 1.02964e+17 59280.1 1e-11 0.99654 0.00345962 2.26225e-21 0 1.02964e-11
+1000 3.45602e+19 6.89205e+08 1e-11 1.29846e-06 0.00363514 0.0654305 0.930933 3.45602e-09
```

Comment:
So at low temperatures the changes lead to higher viscosities and stresses. Is that because of the power law of the drucker prager rheology? Do you approach the old values as your stress exponent goes to infinity? I just want to make sure the difference we see is caused by the functional change in the drucker prager rheology, and not by the change to the Newton iteration.

Reply:
Yes and yes. 

- As the temperature goes down, the purely viscous stress at a given strain rate increases. 
- The "yield stress" in the new Drucker Prager flow law is associated with a constant reference strain rate
- At a fixed strain rate, the ratio of plastic strain to viscous strain will increase as temperature decreases.

At even lower temperatures, plastic strain increasingly will dominate, and so the stress will tend towards a constant value. That value will depend on the "yield stress", reference strain rate and actual strain rate.